pkg/services/sqlstore/database_wrapper.go

```diff
@@ -100,10 +101,11 @@ func (h *databaseQueryWrapper) instrument(ctx context.Context, status string, qu
 	_, span := h.tracer.Start(ctx, "database query", trace.WithTimestamp(begin))
 	defer span.End()
 
-	span.AddEvents([]string{"query", "status"}, []tracing.EventValue{{Str: query}, {Str: status}})
+	span.AddEvent("query", trace.WithAttributes(attribute.String("query", query)))
+	span.AddEvent("status", trace.WithAttributes(attribute.String("status", status)))
 
 	if err != nil {
-		span.AddEvents([]string{"error"}, []tracing.EventValue{{Str: err.Error()}})
+		span.RecordError(err)
```

Comment:
This will change to one event with name `exception` and attributes of `exception.type` and `exception.message`, ref https://github.com/open-telemetry/opentelemetry-go/blob/sdk/v1.16.0/sdk/trace/span.go#L427-L445

Reply:
LGTM!